src/components/PositionsList/Pagination/Pagination.tsx

```diff
@@ -7,13 +7,19 @@ export interface IProps {
   pages: number
```

Comment:
maybe move Pagination out of position list folder

Reply:
done